src/libs/Localize/index.ts

```diff
@@ -106,11 +123,21 @@ function getTranslatedPhrase<TKey extends TranslationPaths>(
         return valueFromCache;
     }
 
-    const translatedPhrase = translations?.[language]?.[phraseKey] as Phrase<TKey>;
+    const translatedPhrase = translations?.[language]?.[phraseKey];
 
     if (translatedPhrase) {
         if (typeof translatedPhrase === 'function') {
-            return translatedPhrase(...phraseParameters);
+            const calledTranslatedPhrase = translatedPhrase(phraseParameters[0]);
+            if (typeof calledTranslatedPhrase === 'string') {
+                return calledTranslatedPhrase;
+            }
+            const count = phraseParameters[1] ?? 0;
```

Comment:
Why are we setting the count default to 0 if there is a first element?

Reply:
What I am trying to say is that we should not add the fallback to 0. If there is no count, then we can return the translated phrase, right?